frontend/src/lib/candidate/components/PasswordField/PasswordField.svelte

```diff
@@ -32,9 +32,9 @@ to reveal and hide the password
 -->
 
 <div class="relative">
-  <label for="password" class="hidden">{$t('candidate.password')}</label>
+  <label for={password} class="hidden">{$t('candidate.password')}</label>
```

Comment:
I'd change this that the id could be given as props with a default value of "password" as now the actual password the user is typing is used for the id field